### DISCUSSION TITLE
what are u doing here?

### DISCUSSION THREAD

**Discussion by ran123123123** [ran123123123/Template#1] [Announcements]
-  👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

<!--
  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to

**Comment by MackDing** [comment]
can you  mark me?

**Reply by ran123123123** [reply]
yes of course x

**Reply by ran123123123** [reply]
mark?

**Reply by MackDing** [reply]
hoho  all  right

**Comment by ran123123123** [comment]
> can you mark me?

https://github.com/ran123123123/Template/discussions/1#discussioncomment-6739728

**Reply by MackDing** [reply]
酱? 我还能说什么呢

**Comment by MackDing** [comment]
奇葩说  酱? 我还能说什么呢

**Comment by ran123123123** [comment]
okkk

**Comment by MackDing** [comment]
> mark?
